Service/Runner.php

```diff
@@ -260,6 +271,8 @@ public function runComponent($jobId, $configId, Component $component)
         if ($this->shouldStoreState($component->getId(), $configId)) {
             $this->stateFile->storeStateFile();
         }
+        $this->usageFile->storeUsage();
```

Comment:
Dve veci 1) usage by se podle me mel logovat i v pripade chyby, napr. pokud se bude platit za api cally, tak je celkem fuk, ze to pak spadne, ale zalogovat (a zaplatit) se to musi
2) pokud se to ulozi do /data/out/ tak se to pri presune na /data/in/, pokud bude mit komponenta (post)procesory, mozna by bylo lepsi misto Runner.php:322 za `$componentOutput = $output->getOutput();` ?